raiden_contracts/tests/test_print_gas.py

```diff
@@ -260,6 +262,15 @@ def print_gas_monitoring_service(
     )
     service_registry.functions.deposit(SERVICE_DEPOSIT).call_and_transact({"from": MS})
 
+    # B registers itself as a service provider
+    deposit = service_registry.functions.currentPrice().call()
```

Comment:
Maybe you can refactor this out and use it in `create_service_account` as well to reduce duplication.

Reply:
Turns out these lines can simply be removed.  It's all done in `create_service_account` above.